src/scripts/main.js

```diff
@@ -1,3 +1,354 @@
 'use strict';
 
-// write code here
+const mainPage = document.querySelector('body');
+
+let table = document.querySelector('tbody');
+
+const header = document.querySelector('thead').lastElementChild;
+
+let rows = [];
+
+const successTitle = 'Success';
+const successMessage = 'New employee record has been added successfully';
+const errorTitle = 'Error';
+let errorMessage = '';
+
+function sortAsc(i) {
+  rows = [...document.querySelector('tbody').children].sort((a, b) =>
+    a.children[i].innerHTML
+      .localeCompare(b.children[i].innerHTML));
+
+  return rows;
+}
+
+function sortDesc(i) {
```

Comment:
you can use one method instead of two, just create a variable to save sorted values or not, and depends on that variable sort Asc or Desc